packages/react-components/src/button/src/Button.css

```diff
@@ -56,22 +57,22 @@ a.o-ui-button {
 
 /* SOLID */
 .o-ui-button-solid {
-    color: var(--white);
-    background-color: var(--marine-500);
+    color: var(--static-white);
+    background-color: var(--foreground-color-5);
```

Comment:
I am not sure as a developper how I will understand this code 1-2-3-4-5 it's starting to be a lot.

I could manage with 1 or 2 variations but 5 is too many.

Reply:
There is no logic, it's months of design piling up. A cleaning could be done with the product designers.